getting-started/using-the-sdk.md

```diff
@@ -91,16 +90,16 @@ Using the REPL:
 
 ```js
 // Create or load account
-const mnemonic = loadOrCreateMnemonic("fred.key");
-mnemonicToAddress("cosmos", mnemonic)
+const mnemonic = loadOrCreateMnemonic('fred.key')
```

Comment:
These are auto-formatting applied by vscode. I was lazy to revert them, and they do no harm.